vrk-kpa/suomidigi
# SUOPA-786 Events

To test: 
- `make fresh drush-cr`
- Log in and go to create at least 3 events. https://suomidigi.docker.sh/node/add/event
   - Fill in at least "Otsikko", Tapahtuma-aika (either in the past or in the future), Kuvaus, Tapahtumapaikka: Paikan nimi and osoitetiedot
- After all of the events have been added, go to create a landing page. https://suomidigi.docker.sh/node/add/landing_page
   - Give the page a title "Tapahtumat" and add a Views (Näkymä) paragraph. Select "Sisältösivu: Tapahtumat" for the view and "Menneet/Tulevat tapahtumat | Ruudukko" for the display - depending of whether the events you created are in the past or in the future
   - Save the node and check that the layout is similar to https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/e54bb2d2-3803-4a7a-b434-598f3b291231/Suomidigi-2020-tapahtumat-XL

If all is good, go to edit the front page https://suomidigi.docker.sh/node/7/edit 
- Add new Views paragraph.
   - Give it a Title "Tapahtumat"
   - Select "Nostot: Sisältö" for the view and "Menneet/Tulevat tapahtumat | Ruudukko" for the display
   - Add description
   - Add link to "Tapahtumat" page you created earlier
- Save the front page 
- Check that the layout is similar to the one in proto: https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/
